exercises/practice/yacht/.approaches/functions/content.md

```diff
@@ -0,0 +1,82 @@
+## Approach: functions
+Each bit of functionality for each category can be encoded in a function, and the constant set to that function. 
```

Comment:
```suggestion
Each bit of functionality for each category can be encoded in an anonymous function (otherwise known as a [`lambda`](https://docs.python.org/3/howto/functional.html?highlight=lambda#small-functions-and-the-lambda-expression)), and the constant name set to that function. 
```